### DISCUSSION TITLE
Visualizing typespec models

### DISCUSSION THREAD

**Discussion by cincauhangus** [microsoft/typespec#4475] [Ideas]
In a large spec project, it would be difficult to visualize the entire model. If there's a way to emit only the models into a class or ER diagram, it would greatly help to visualize the data model.

**Comment by timotheeguerin** [comment]
We don't have this diagram emitter though this could be a nice addition
the closet thing you can do today is open your spec in the playground and navigate to the model in teh type graph viewer
